learn/how-tos/implement-openid-google-provider.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Configuring Open ID using Google provider"
+id: "implement-openid-google-provider"
+sidebar_label: "Open ID using Google provider"
+---
+In this document we discuss the steps to implement different open ID providers.
+
+---
+
+## Google as OpenID provider
```

Comment:
We should add a heading on how to further configure Google as openID provider:

1.  Retrieve profile information upon successful login
2.  Restrict allowed domains